src/tablo/Tablo.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import cx from 'classnames';
+import t, { maybe } from 'tcomb';
+import { props  } from 'tcomb-react';
+import { pure, skinnable } from 'revenge';
+import { Table } from 'fixed-data-table-2';
+import Column, { defaultColumns, updateColumns } from './Column';
+import FlexView from 'react-flexview';
+import { autosize } from './plugins';
+
+export const defaultWidth = 500;
+
+/** A table component based on fixed-data-table-2
+ * @param data - data shown in the table
+ * @param defaultColumns - a function that returns the default columns given the data
+ * @param width - the desired width of the table
+ * @param height - the desired height of the table
+ * @param rowHeight - height in pixel of every row
+ * @param headerHeight - height in pixel of header
+ * @param groupHeaderHeight - height in pixel of groupHeader
+ * @param footerHeight - height in pixel of footer
+ * @param children - content
+ */
+@autosize
+@skinnable()
+@pure
+@props({
+  className: maybe(t.String),
+  data: t.Array,
+  defaultColumns: maybe(t.Function),
+  width: t.Number,
+  height: t.Number,
+  rowHeight: t.Number,
+  headerHeight: t.Number,
+  groupHeaderHeight: t.Number,
+  footerHeight: t.Number,
+  children: t.ReactChildren
+})
+export default class Tablo extends React.Component {
+
+  static defaultProps = {
+    width: defaultWidth,
```

Comment:
While I agree to remove `width` and `height` as default props (they are actually totally random) I don't think the default props are to be avoided as they are evil.
For instance, for me a default row height makes sense in a table component and I'd blame the dev if he forgets to pass the prop explicitly when required by specs.

Anyway what happens here is that actually `width` and `height` are ignored unless one doesn't pass `autosize={false}` (autosizing future turned out to be extremely helpful in all the usages of the tables in our projects)